src/main/java/su/litvak/chromecast/api/v2/Channel.java

```diff
@@ -380,7 +380,7 @@ private void connect() throws IOException, GeneralSecurityException {
             if (response instanceof StandardResponse.Invalid) {
                 StandardResponse.Invalid invalid = (StandardResponse.Invalid) response;
                 throw new ChromeCastException("Invalid request: " + invalid.reason);
-            } else if (response instanceof StandardResponse.LoadFailed) {
+            } else if (response instanceof StandardResponse.LoadCancelled || response instanceof StandardResponse.LoadFailed) {
```

Comment:
I think it should fall under the different `if` branch, throwing an error saying that the Load was cancelled. However I am not sure about the use case.